context/app/static/js/components/search/Results/ResultsTiles.tsx

```diff
@@ -97,6 +97,7 @@ function Tile({ hit }: { hit: SearchHit<Partial<Entity>> }) {
       uuid={hit?._source?.uuid}
       id={hit?._source?.hubmap_id}
       entityData={hit?._source}
+      ariaLabelText={`Tile representing ${hit?._source?.entity_type} ${hit?._source?.uuid}`}
```

Comment:
I think the HBM ID would be preferable to the UUID here since the screenreader would have to read off all 32 characters of the UUID.

We may also want to be more descriptive depending on the result type - e.g. right now this will say `Tile representing Dataset blahblahblah` or `Tile representing Sample blahblahblah`, but in the future we could provide e.g. the assay type and source organ for datasets, sample category and source organ for samples, and descriptive info for donors.